ethereum/helpers/uris.ts

```diff
@@ -9,15 +9,15 @@ export function getURITemplate(
+        `select json_object('name','Rig #'||rig_id,'external_url','https://tableland.xyz/rigs/'||rig_id,'image','ipfs://'||renders_cid||'/'||rig_id||'/'||image_full_name,'image_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_full_alpha_name,'image_medium','ipfs://'||renders_cid||'/'||rig_id||'/'||image_medium_name,'image_medium_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_medium_alpha_name,'thumb','ipfs://'||renders_cid||'/'||rig_id||'/'||image_thumb_name,'thumb_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_thumb_alpha_name,'animation_url',animation_base_url||rig_id||'.html','attributes',json_array(json_object('trait_type','status','value','pre-reveal'))) from ${attributesTable} join ${lookupsTable} where rig_id=ID group by rig_id;`
       );
     return uri.split("ID");
   } else {
     const uri =
       tablelandHost +
       "/query?extract=true&unwrap=true&s=" +
       encodeURIComponent(
-        `select json_object('name','Rig #'||rig_id,'external_url','https://tableland.xyz/rigs/'||rig_id,'image','ipfs://'||renders_cid||'/'||rig_id||'/'||image_full_name,'image_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_full_alpha_name,'image_medium','ipfs://'||renders_cid||'/'||rig_id||'/'||image_medium_name,'image_medium_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_medium_alpha_name,'thumb','ipfs://'||renders_cid||'/'||rig_id||'/'||image_thumb_name,'thumb_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_thumb_alpha_name,'animation_url',animation_base_url||rig_id,'attributes',json_group_array(json_object('display_type',display_type,'trait_type',trait_type,'value',value))) from ${attributesTable} join ${lookupsTable} where rig_id=ID group by rig_id;`
+        `select json_object('name','Rig #'||rig_id,'external_url','https://tableland.xyz/rigs/'||rig_id,'image','ipfs://'||renders_cid||'/'||rig_id||'/'||image_full_name,'image_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_full_alpha_name,'image_medium','ipfs://'||renders_cid||'/'||rig_id||'/'||image_medium_name,'image_medium_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_medium_alpha_name,'thumb','ipfs://'||renders_cid||'/'||rig_id||'/'||image_thumb_name,'thumb_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_thumb_alpha_name,'animation_url',animation_base_url||rig_id||'.html','attributes',json_group_array(json_object('display_type',display_type,'trait_type',trait_type,'value',value))) from ${attributesTable} join ${lookupsTable} where rig_id=ID group by rig_id;`
```

Comment:
In both changes here, note and validate the updated `animation_url` value `animation_base_url||rig_id||'.html'`

Reply:
I think it would be easier to revert these changes and just take what's on `add-pilots` branch when that lands. I have done some good formatting and also added a synthetic trait for "piloted": https://github.com/tablelandnetwork/rigs/blob/add-pilots/ethereum/helpers/uris.ts

Reply:
Oh nice, yea much better that way. Reverting. Did you do the same thing in the tests?

Reply:
yep, tests are updated there

Reply:
Ok @sanderpick i reverted and pushed. Give me an approval here if we're good to go pls.